File: C++/bronze/2738.cpp
#include <iostream>
#include <vector>

using namespace std;

int main(){
    int N, M, temp;
    cin >> N >> M;
    vector<vector<int>> matrix(N, vector<int>(M, 0));
    for (int k = 0; k < 2; k++){
        for (int i = 0; i < N; i++){
            for (int j = 0; j < M; j++){
                cin >> temp;
                matrix[i][j] += temp;
            }
        }
    }
    for (int i = 0; i < N; i++){
            for (int j = 0; j < M; j++){
                cout << matrix[i][j] << " ";
            }
            cout << endl;
        }
    return 0;
}

File: python/bronze/2738.py
N, M = map(int, input().split())
matrix = [[0]*M for _ in range(N)]
for k in range(2):
    for i in range(N):
        for j, temp in enumerate(map(int, input().split())):
            matrix[i][j] += temp
for row in matrix:
    print(*row)